routes/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = app => {
+
+    // Base URLS
+    app.use('/', require('./base.routes.js'))
+    app.use('/', require('./auth.routes'))
```

Comment:
```suggestion
    app.use('/', require('./base.routes.js'))
    app.use('/auth', require('./auth.routes'))
```
El problema es que tenías dos endpoints que usaban el mismo path (_/empleados_) y el mismo método HTTP (_GET_), cada uno en archivos diferentes, pero al comenzar cada path para cada archivo igual, entraba por el primero, en este caso el que está más arriba y nunca llegaba al que tu querías. Es buena idea usar para cada archivo de rutas un path diferente y así evitas este problema.

Entiendo que te sientas frustrado @gconde-m2 pero este laboratorio se suele atravesar bastante, es largo, tenéis el tiempo muy ajustado y ya se va notando el cansancio del bootcamp.

Ánimo y sigue asi que vas super bien!